components/Modal/index.tsx

```diff
@@ -0,0 +1,31 @@
+import Swal from 'sweetalert2'
+
+export default function Modal(str: string, callback: any) {
+  return Swal.fire({
+    allowOutsideClick: () => !Swal.isLoading(),
+    backdrop: 'rgba(0, 0, 0, 0.7)',
+    confirmButtonText: 'Aceptar',
+    input: 'text',
+    inputValidator: value => {
+      if (!value) {
+        return 'Ingresa una palabra'
+      } else if (value !== 'ELIMINAR') {
+        return 'Palabra incorrecta'
+      }
+    },
+    inputPlaceholder: 'ELIMINAR',
+    padding: '50px 120px 40px',
+    showCancelButton: true,
+    html: '<p>Si desea eliminar este, escriba <span>ELIMINAR</span> en el siguiente recuadro.</p>',
```

Comment:
Why not use this?
https://sweetalert2.github.io/recipe-gallery/sweetalert2-react.html

Reply:
Thank you very much. Yes, that seems to be the problem. I didn't implement it because I needed some functionality that sweetalert-react didn't have. But I think it's going well